portsyncd/linksync.cpp

```diff
@@ -39,18 +39,21 @@ struct if_nameindex
     unsigned int if_index;
     char *if_name;
 };
-extern "C" { extern struct if_nameindex *if_nameindex (void) __THROW; }
+extern "C" {
+    extern struct if_nameindex *if_nameindex (void) __THROW;
+    extern void if_freenameindex (struct if_nameindex *) __THROW;
```

Comment:
>if_freenameindex [](http://example.com/codeflow?start=16&length=16)

Why not just include header file and remove all the `extern` lines?
```
#include <net/if.h>
```

Reply:
Good idea, fixed in https://github.com/Azure/sonic-swss/pull/2221/commits/737d6d4724d2be7b5825b6efa053d35b1688957d